.github/workflows/build-test-macos-blockchain.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     steps:
     - name: Cancel previous runs on the same branch
       if: ${{ github.ref != 'refs/heads/main' }}
-      uses: styfle/cancel-workflow-action@0.9.1
```

Comment:
I think you should update the template with version `0.9.1`, then all of these won't regress

Reply:
Good point, changed! I didn't even realized that this are downgrades :see_no_evil: